src/System.Net.Sockets/src/System/Net/Sockets/_BaseOverlappedAsyncResult.Unix.cs

```diff
@@ -10,18 +10,12 @@
 
 namespace System.Net.Sockets
 {
+    // BaseOverlappedAsyncResult
```

Comment:
Nit: I think we should remove the line above: it's just the name of the class below.